### PR TITLE
Jeremy/fix map bugs

### PR DESCRIPTION
**Pull request by lonerly666**
### Summary
Fixes some bugs on the map

### Details
There are some bugs in the map:
- No auto zoom when searching for locations that need to zoom 
- No updates on travel guides and itineraries when using search bar

### CODE REVIEW

File: frontend/screens/Map.js [hunk 1]
```diff
@@ -405,8 +430,8 @@ const Map = () => {
           setRegion({
             latitude: data.geometry.location.lat,
             longitude: data.geometry.location.lng,
-            latitudeDelta: region.latitudeDelta,
-            longitudeDelta: region.longitudeDelta,
+            latitudeDelta: 0.0922,
```
Comment (TadasGliadkovskis) [inline frontend/screens/Map.js:433]: why is this numbered?

**Review by TadasGliadkovskis — APPROVED** [review]
LGTM Nice Fixes

File: frontend/screens/Map.js (hunk 1 above)
Comment (TadasGliadkovskis) [reply frontend/screens/Map.js:433]: onPoiClick does not gives us the delta values
and the delta values are the ones that help the map to zoom
Therefore fixing the issue of when we click on a location the map will automatically zoom into it

**Review by Denniswillie — APPROVED** [review]

**Review by TadasGliadkovskis — APPROVED** [review]
LGTM! Careful with creating more bugs 😂

**Pull request merged**